src/query/management/src/role/role_mgr.rs

```diff
@@ -158,7 +157,13 @@ impl RoleApi for RoleMgr {
             res.ok_or_else(|| ErrorCode::UnknownRole(format!("Role '{}' does not exist.", role)))?;
 
         match seq.match_seq(&seq_value) {
-            Ok(_) => Ok(seq_value.into_seqv()?),
+            Ok(_) => {
+                let data = serde_json::from_slice::<RoleInfo>(&seq_value.data).or_else(|err| {
+                    log::debug!("serde json err when get role. cause : {}", err);
+                    deserialize_struct(&seq_value.data, ErrorCode::IllegalUserInfoFormat, || "")
```

Comment:
Please add a comments for this, it's not by design, it's for compatible with the old version.